restAPI/controllers/controller_auth.js

```diff
@@ -0,0 +1,58 @@
+const express = require("express");
+/* eslint-disable-next-line */
+const router = express.Router();
+/* eslint-disable-next-line */
+const logger = require("../../logger").logger;
+const { DATABASE, COLLECTION } = require("../../public/constant");
+const jwt = require("jsonwebtoken"); // import jwt to sign tokens
+
+/**
+ * @async
+ * @route   GET /api/v1/auth/testAPI
+ * @returns {object} test API
+ * @author  Hassan Ali
+ * @access  public
+ * @version 1.0
+ */
+
+exports.testAPI = async (req, res, next) => {
+  res.status(200).send({
+    error: false,
+    code: 200,
+    status: "succsess",
+    message: "Test API Works",
+  });
+};
+
+// DB connection
+const mongo_conn_native = require("../../mongo_conn_native").Connection;
+
+/**
+ * @async
+ * @route       Post /api/v1/auth/:id
+ * @returns     {Users} user object
+ * @author      Hassan Ali
+ * @access      public
+ * @version     1.0
+ */
+
+exports.login = async (req, res, next) => {
+  const client = mongo_conn_native.client;
+  const { userName, password } = req.body;
```

Comment:
```suggestion
  const { _username , _password } = req.body;
```
For local vars I'd recommend to use `_` underscore in the beginning to make sure it will be be used outside the scope.